README.md

```diff
@@ -2,6 +2,17 @@
   <img src="https://media.onesignal.com/cms/Website%20Layout/logo-red.svg"/>
 </p>
 
+#### ⚠️ Migration Advisory for current OneSignal customers
```

Comment:
Let's put this under the **OneSignal Android Push Notification Plugin** heading?